tests/mapped_struct.py

```diff
@@ -987,21 +987,33 @@ def _testBuild(self, N, tempdir, **gen_kwargs):
             if v not in elem:  # Purposefully testing not in operator
                 self.assertIn(v, elem)
 
-            int_elem = rvget(k)
-            self.assertIsNotNone(int_elem)
-
     # Too much memory
     testBuildHugeInMemShuffled = None
     testBuildHugeOnDiskShuffled = None
 
-class ApproxStringId32MultiMapperTest(ApproxStringIdMultiMapperTest):
+class ApproxStringId32MultiMapperTest(IdMapperTest):
     IdMapperClass = mapped_struct.ApproxStringId32MultiMapper
```

Comment:
What's up with this? This is wrong, you're changing an existing and working test

Reply:
Ups.